Sources/Clock.swift

```diff
@@ -32,7 +44,11 @@ public struct Clock {
 
     /// The most accurate date that we have so far (nil if no synchronization was done yet)
     public static var now: Date? {
-        return self.timestamp.map { Date(timeIntervalSince1970: $0) }
+        return self.fullNow?.date
+    }
+
+    public static var fullNow: FullTime? {
```

Comment:
missing docstring, and same comment as below, I'd expose this as `timeSinceLastNTPSync`

Reply:
can't do this bc it's racy, explanation below